build.gradle

```diff
@@ -1,3 +1,6 @@
+def configDocDir = "${buildDir}/configDocs"
```

Comment:
I am wondering if it may be better to have the doc under ${project.rootDir}/docs instead of ${buildDir}. User typically would ignore the content under build directory. On the other hand, the docs under root directory seems more obvious. Apache Kafka generated docs under ${project.rootDir}/docs/generated.